WP_Auth0.php

```diff
@@ -493,6 +492,16 @@ public static function get_plugin_dir_url() {
 	}
 }
 
+/*
+ * Startup
+ */
+
+function wp_auth0_db_check_update() {
```

Comment:
shouldn't this change be documented on the init() function or somewhere else?

Reply:
`init()` was removed